tests/metrics/test_faithfulness_metrics.py

```diff
@@ -411,7 +411,7 @@ def test_faithfulness_estimate(
                 "disable_warnings": True,
                 "display_progressbar": True,
             },
-            {"min": 0.0, "max": 1.0},
+            {"min": 0.0, "max": 80.0},
```

Comment:
can we make it explicit in the text if we use softmax or not? so that min and max make sense?